src/web/utils/__tests__/firefoxUtils.test.ts

```diff
@@ -0,0 +1,72 @@
+import * as CursorUtils from '../cursorUtils';
+import type {MarkdownTextInputElement} from '../../../MarkdownTextInput.web';
+import {handleFirefoxRightArrowKeyNavigation} from '../firefoxUtils';
+
+const createMockTarget = (value: string): MarkdownTextInputElement => {
+  const div = document.createElement('div') as unknown as MarkdownTextInputElement;
+  div.value = value;
+  return div;
+};
+
+jest.mock('../cursorUtils', () => ({
+  ...jest.requireActual('../cursorUtils'),
+  getCurrentCursorPosition: jest.fn(),
+  setCursorPosition: jest.fn(),
+}));
+
+describe('handleFirefoxArrowKeyNavigation', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
Let's add a newline here
```suggestion
  });

```